cellprofiler/modules/measuregranularity.py

```diff
@@ -1,12 +1,14 @@
 # coding=utf-8
 
 """
-**Measure Granularity** outputs spectra of size measurements of the
+MeasureGranularity
+==================
+**MeasureGranularity** outputs spectra of size measurements of the
```

Comment:
At the end of this sentence please add the sentence  "This module supports 2D and volumetric images"